Makefile

```diff
@@ -13,12 +13,23 @@
 
 all: release
 
+# Improve buildspeeds during development by removing the --no-cache flag
+DOCKER_BUILD_FLAGS=--no-cache
+
 .PHONY: release
-release: linux-plugins
+release: build
 	docker system prune -f
-	docker build -t amazon/aws-for-fluent-bit:latest -f Dockerfile .
+	docker build $(DOCKER_BUILD_FLAGS) -t amazon/aws-for-fluent-bit:latest -f Dockerfile.vanilla-release .
+	docker system prune -f
+	docker build $(DOCKER_BUILD_FLAGS) -t amazon/aws-for-fluent-bit:init-latest -f Dockerfile.init-release .
+
+.PHONY: debug
```

Comment:
this issue noted that we need to update debugging guide when we have a finalized this, since you can't just build the dockerfiles anymore: https://github.com/aws/aws-for-fluent-bit/issues/579